ui/src/shared/utils/vis.ts

```diff
@@ -120,6 +120,10 @@ export const parseBounds = (
 }
 
 export const extent = (xs: number[]): [number, number] | null => {
+  if (!xs || !xs.length) {
```

Comment:
what does this do?

Reply:
even though extent is always supposed to be passed a number array, it doesn't when an empty state is loaded from redux. i thought about guarding it at the redux retrieval with a default value, but it seemed that if could happen there, it could probably happen from anywhere and put the guard as low as possible (where it breaks).